### PR TITLE
Update the Shipyard test framework

### PR DESCRIPTION
**Pull request by skitt**
... and adjust to changes.

Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

**Comment by MaxBab** [comment]
@skitt 

Hi Stephen,

My patch that fixes exactly this is waiting for review.
@tpantelis requested small changes and I already did them.

Could you take a look at the patch, please?
If you find the patch ok and a bit later today @tpantelis approves it as well, we could merge it.

https://github.com/submariner-io/submariner/pull/2014

**Review by mkolesnik — APPROVED** [review]
FWIW I think we should merge this if it passes to unblock the rc0 release, and review #2014 without rushing

**Review by tpantelis — APPROVED** [review]

**Comment by skitt** [comment]
Backported in #2083.

**Pull request merged**
